lib/withdrawal_tokens/widgets/withdrawal_address_input.dart

```diff
@@ -37,13 +37,14 @@ class _WithdrawalAddressInput extends StatefulWidget {
   final TextEditingController? withdrawalAddressController;
 
   @override
-  State<_WithdrawalAddressInput> createState() =>
-      _WithdrawalAddressInputState();
+  State<WithdrawalAddressInputPage> createState() =>
+      WithdrawalAddressInputPageState();
 }
 
-class _WithdrawalAddressInputState extends State<_WithdrawalAddressInput> {
-  final TextEditingController withdrawalAddressController =
-      TextEditingController();
+class WithdrawalAddressInputPageState
```

Comment:
@bibash28  It's better to be private also

Reply:
yes we may make it private...